scripts/js/upgrade/index.js

```diff
@@ -4,8 +4,17 @@ const fs = require('fs')
 const util = require('util');
 const exec = util.promisify(require('child_process').exec);
 
-const PREIMAGE_LENGTH_BOUND = 34;
-const PROPOSAL_LENGTH_BOUND = 90;
+// Needs to be >= 34
+// 32 bytes from the encoding of the H256 hashed WASM blob
+// 2 for extra stuff
+const AUTHORIZE_UPGRADE_PREIMAGE_LENGTH_BOUND = 34;
+// Needs to be >= 84
+// 39 from edemocracy.xternalProposeMajority(Lookup(H256, 34)))
+// 42 from democracy.fastTrack(H256, ...)
+// 1 from utility.batchAll
+// 2 extra
```

Comment:
I guess the "2 extra" bytes is the first for the pallet index and the second for the extrinsic index?

Reply:
[bf413ad](https://github.com/centrifuge/centrifuge-chain/pull/1308/commits/bf413adb428b6b71903079086c076ac122924df1)